packages/ui/src/index.tsx

```diff
@@ -83,7 +83,7 @@ import Tabs from "./components/Tabs";
 import ToggleGroup from "./components/ToggleGroup";
 import ToggleSection from "./components/ToggleSection/ToggleSection";
 import TreeView from "./components/TreeView";
-import Well, { WellProps } from "./components/Well";
+import Well, { WellProps, WellColor } from "./components/Well";
```

Comment:
Do we need to export this? I don't tihnk I see it being used outside of this package

Reply:
I was using it in StatusSummary.tsx, however I forgot to switch it back to use Color after refactoring so I can remove these exports now :+1:

Reply:
Changed in [d0158a5](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/d0158a5dede6229a8a5ec42b248ce3b817d869b8)